js/user.js

```diff
@@ -1,24 +1,16 @@
-(function () {
-    let sidebar = document.getElementById("content-left");
-    let loadCommonCourses = null;
-
-    if (sidebar) {
-        let button = createButton("splus-user-courses-in-common-btn", "Courses in Common");
-        let img = createElement("img", [], { src: chrome.runtime.getURL("imgs/plus-icon.png"), width: 18, style: { verticalAlign: "middle", paddingLeft: "4px" } });
-        button.prepend(img);
-        button.querySelector("input").style.paddingLeft = "4px";
-        button.style.cursor = "pointer";
-        button.addEventListener("click", () => {
-            if (loadCommonCourses == null) {
-                loadCommonCourses = getCoursesInCommon(document.location.href.match(/\/(\d+)\//)[1]);
-            }
-            openModal("user-courses-in-common-modal", { loadCommonCourses: loadCommonCourses, userId: document.location.href.match(/\/(\d+)\//)[1] });
-        });
-
-        sidebar.appendChild(button);
-    }
+(async function () {
+    const page = document.getElementById("main-inner");
+    if (!page) return;
+    const userID = document.location.href.match(/\/(\d+)\//)[1];
+    const loadCommonCourses = getCoursesInCommon(userID);
+    const container = createElement("div", ["splus-modal-contents"], {}, [
+                        createElement("ul", ["setting-entry", "common-realm-list"], { id: "user-courses-in-common-list" }, [])
+                    ]);
+    const title = createElement("h4", [], {textContent: "Courses In Common"});

```

Comment:
Stylistically, can we make this closer to a "proper" section? See this screenshot (Firefox):
![image](https://user-images.githubusercontent.com/3952667/93833058-674bc180-fc2c-11ea-9e19-34b68d66f419.png)
Can we have "Courses In Common" be closer stylistically to "About Me" or "Contact Information"?